src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/SentenceCalcTypeRepository.java

```diff
@@ -8,7 +8,7 @@
 
 public interface SentenceCalcTypeRepository extends CrudRepository<SentenceCalcType, SentenceCalcType.PK> {
 
-    @Cacheable("calculable_sentence_calculation_types")
```

Comment:
calling this previously resulted in `Cannot find cache named 'calculable_sentence_calculation_types'` error